tensorflow/core/kernels/data/dataset_test_base.cc

```diff
@@ -895,5 +905,106 @@ std::vector<FunctionDef> MapDatasetParams::func_lib() const {
   return func_lib_;
 }
 
+TensorSliceDatasetParams::TensorSliceDatasetParams(
```

Comment:
I thought we agreed to only put shared `*DatasetParams` in this class. Are these currently (expected to be) shared?

Reply:
`TensorSliceDatasetParams` are used in `CacheDatasetOpTest` and `ConcatenateDatasetOpTest`; `TakeDatasetParams` is used by `AssertNextDatasetOpTest`.